MH-FiorellaTheme5/source/javascripts/site.js

```diff
@@ -1 +1,12 @@
 // This is where it all goes :)
+function openNav() {
+    document.getElementById("mySidenav").style.width = "95%";
+    document.getElementById("logo").src = "../images/home/logo_fiorella_menu.png";
```

Comment:
Line is too long.

Reply:
the JS style recommends to use single quotes instead double quotes.